vizro-core/src/vizro/actions/_callback_mapping/_callback_mapping_utils.py

```diff
@@ -97,7 +97,7 @@ def _get_action_callback_outputs(action: Action) -> dict[str, Output]:
         targets = []
 
     if action_function == _parameter.__wrapped__:
-        targets = [target.split(".")[0] for target in targets]
+        targets = [target.split(".")[0] if "." in target else target for target in targets]
```

Comment:
Please can you put a comment in here explaining that the case without `"."` is just for the case of dynamic data parameter affecting a filter?